src/local-editor-service.ts

```diff
@@ -85,12 +85,11 @@ export class LocalEditorService extends EditorService {
       return {
         kind: 'element-tags',
         elements: elements.map(e => {
-          const attributesSpace = e.attributes.length > 0 ? ' ' : '';
           return {
             tagname: e.tagName!,
             description: e.description,
             expandTo: location.kind === 'text' ?
-                `<${e.tagName}${attributesSpace}></${e.tagName}>` :
+                this._generateAutoCompletionForElement(e) :
```

Comment:
So the language server protocol folks are exploring cursor positioning with completions. Open bug here: https://github.com/Microsoft/language-server-protocol/issues/58

The text based format is fine for now, but it leaves us with some problems, like how it's hard to explain to the editor whether it should insert a `<` or not (i.e. https://github.com/Polymer/atom-plugin/issues/2), or if the user has already typed `<paper-button>` and then they autocomplete on `<paper-button></paper-button>` do we want to just add the closing tag or is this a new tag?

The text-based format leaves all of that decision making up to the editor, even when we have more information. So the LSP is moving towards something more explicit, where you can specify exactly the transformation of the document that should occur, and exactly where and how to position and tab through the cursor(s) afterwards.

(All of that is just information for the future, this PR is great and definitely improves things)